cmd/av/stack_branch.go

```diff
@@ -104,3 +111,63 @@ var stackBranchCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func init() {
+	// pass
+	stackBranchCmd.Flags().StringVar(&stackBranchFlags.Parent, "parent", "", "the parent branch to base the new branch off of")
+	stackBranchCmd.Flags().BoolVarP(&stackBranchFlags.Move, "move", "m", false, "if true, move/rename the current branch")
+}
+
+func stackBranchMove(repo *git.Repo, newBranch string) error {
+	oldBranch, err := repo.CurrentBranchName()
+	if err != nil {
+		return err
+	}
+
+	if oldBranch == newBranch {
+		return errors.Errorf("cannot rename branch to itself")
+	}
+
+	currentMeta, _ := meta.ReadBranch(repo, oldBranch)
+	currentMeta.Name = newBranch
+	if err := meta.DeleteBranch(repo, oldBranch); err != nil {
+		return err
+	}
+	if err := meta.WriteBranch(repo, currentMeta); err != nil {
+		return err
+	}
+
+	if !currentMeta.Parent.Trunk {
```

Comment:
can you add comments describing this part. If I understand this correctly, we are setting the parent to point to the new branch as the child.